src/main/java/seedu/address/logic/commands/BudgetCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.events.ui.ShowBudgetViewEvent;
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.model.tag.TagContainsKeywordPredicate;
+
+
+/**
+ * View the budget of the whole hall or by their CCA.
+ */
+public class BudgetCommand extends Command {
+
+    public static final String COMMAND_WORD = "budget";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": View the budget of the whole hall or by each CCA "
+        + "Parameters: "
+        + "overall or " + PREFIX_TAG + "TAG\n"
+        + "Example: " + COMMAND_WORD + " overall or " + COMMAND_WORD + " basketball\n";
```

Comment:
might want to separate the 2 examples so it's clearer

Reply:
Resolved